src/createServer.js

```diff
@@ -1,11 +1,23 @@
+/* eslint-disable padding-line-between-statements */
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+
+const { userRouter } = require('./routes/user');
+const { expenseRouter } = require('./routes/expense');
+const { resetExpenses } = require('./services/expense');
+const { resetUsers } = require('./services/user');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  resetExpenses();
+  resetUsers();
+
+  app.use('/users', express.json(), userRouter);
```

Comment:
Consider moving 'express.json()' middleware to a global level for the entire app, instead of repeating it for each route.